data/coalition/coalition missions.txt

```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
 				decline
 
 event "alitis freed"
+
+
+
+mission "Saryd University Lecture"
+	name "Saryd Scientists"
+	description "Starlit University on <planet> is hosting a seminar on xenology. Transport a guest speaker there by <date>. Payment is <payment>."
+	passengers 1
+	deadline
+	to offer
+		random < 40 
+		has "Coalition: First Contact: done"
+	source "Warm Slope"
+	destination "Shadow of Leaves"
+	on offer
+		conversation
+			`As you exit from a restaurant in the spaceport, you notice a rumpled-looking Saryd staring at you. This is not particularly unusual for you, as you are quite a strange sight here in Coalition space. However, he then begins to gallop towards you, hailing you down.`
+			`	"Captain! Excited to finally meet you, I am. Heard of the human in our space, I have, but to see you myself, special it is."`
+			`	"A scientist, I am, and invitied to speak at a university seminar, I have been." This time, he speaks in a calmer tone of voice. "Be there by <date>, I must. The honor of transporting us to Shadow of Leaves, will you give me?"`
+			choice
+				`	"What is the seminar about?"`
+					goto "xeno"
+				`	"Sure, I'd be glad to transport you."`
+					goto "accept"
+				`	"Sorry, I can't take transport jobs right now."`
+					decline
+			label "xeno"
+			`	"Funny you ask, it is. Xenologist, I am. Alien culture and physiology, I study, and the topic of the seminar, it is. Invited to give lectures on the subject, I have been. Take me there, will you?"`
+			choice
+				`	"Sure, I'd be glad to transport you."`
+				`	"Sorry, I can't take transport jobs right now."`
+					decline
+			label "accept"
+			`	"Wonderful!" he says. "Aelbris, I am." He soon gathers his things and boards your ship. As he enters, he seems fascinated by its structure and design. His gawking makes it difficult to get him to finally settle down into his bunk.`
+					accept
+	on complete
+		"coalition jobs" += 3
+		payment 232000
+		conversation
+			`During the trip, Aelbris was polite, if a little excited to have a human as his pilot. He asked you several questions during the flight, which you answered as best as you can. Once you and your passenger departs from your ship, Aelbris turns around.`
+			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually meet a human, I have not. A great boon to our field to have a human speak at our seminar, it would be."`
```

Comment:
```suggestion
			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually meet a human, I had not. A great boon to our field to have a human guest speaker, it would be."`
```

Reply:
```suggestion
			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually met a human, I had not. A great boon to our field to have a human guest speaker, it would be."`
```
Just adding a small change to Pointed's suggestion.